Tests/RxCocoaTests/UITextField+RxTests.swift

```diff
@@ -13,35 +13,52 @@ import XCTest
 
 // UITextField
 final class UITextFieldTests : RxTest {
+    
     func test_TextCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.text }
     }
     func test_ValueCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.value }
     }
-
+    
     func testSettingTextDoesntClearMarkedText() {
         let textField = UITextFieldSubclass(frame: CGRect.zero)
-
         textField.text = "Text1"
-        textField.set = false
+        textField.isSettedText = false
         textField.rx.text.on(.next("Text1"))
-        XCTAssertTrue(!textField.set)
+        XCTAssertTrue(!textField.isSettedText)
         textField.rx.text.on(.next("Text2"))
-        XCTAssertTrue(textField.set)
+        XCTAssertTrue(textField.isSettedText)
+    }
+    
+    func testLabel_attributedTextObserver() {
+        let textField = UITextField()
+        XCTAssertEqual(textField.attributedText, "".textFieldAttributedString)
+        let attributedText = "Hello!".textFieldAttributedString
+        _ = Observable.just(attributedText).bind(to: textField.rx.attributedText)
+        XCTAssertEqual(textField.attributedText!, attributedText)
     }
 }
 
-final class UITextFieldSubclass : UITextField {
-    var set: Bool = false
+private extension String {
+    var textFieldAttributedString: NSAttributedString {
+        let tf = UITextField()
+        tf.attributedText = NSAttributedString(string: self)
+        return tf.attributedText!
```

Comment:
I don't understand this code. `UITextField` is being created and then discarded immediately.

Is there something wrong with just using `NSAttributedText(string: "Hello!")`?